tgui/packages/tgui/interfaces/common/WirePanel/Wire.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @copyright 2023
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+import { RemoteAccessButton } from ".";
```

Comment:
I think it can handle it fine, but probably best to extract `RemoteAccessButton` into its own file if it's used here as well.

EDIT: Actually, looks like it's *only* used here, so yeah, bring that out of the index file into its own one and import here.